lib/db/schema.ts

```diff
@@ -65,6 +72,10 @@ export const ClassesTable = pgTable(
     capacity: integer("capacity").notNull(),
     slotsLeft: integer("slots_left").notNull(),
     createdAt: timestamp("created_at").defaultNow().notNull(),
+    startDate: text("start_date").notNull(),
+    time: text("time").notNull(),
+    slotsLeft: integer("slots_left").notNull(),
+    location: text("location").notNull(),
```

Comment:
@hyperleash you can remove these now that my branch is merged

Reply:
Done